sass-transformer.js

```diff
@@ -60,7 +60,7 @@ if ( reactNativeMinorVersion >= 52 ) {
 }
 
 // TODO: need to find a way to pass the include paths and the default asset files via some config
-const autoImportIncludePaths = [ 'gutenberg/edit-post/assets/stylesheets' ];
+const autoImportIncludePaths = [ 'src', 'gutenberg/edit-post/assets/stylesheets' ];
```

Comment:
The need to add `src` seems weird and I feel it's an issue withe the `sass-transformer`. I assume you've added this to enable the `@import '../variables.scss'` statement, right?

Reply:
Pushed de8629d that fixes the issue with the transformer. Follow up 0d2b501 removes the hardcoded import include path.

Reply:
Good job, thanks